package-lock.json

```diff
@@ -1593,7 +1593,7 @@
       "from": "github:Doges/crowdin-node",
       "requires": {
         "bluebird": "^3.7.2",
-        "graceful-fs": "^4.2.4",
+        "graceful-fs": "^4.2.2",
```

Comment:
?

Reply:
Whoops this wasn't intentional, maybe it was auto-edited when I ran `npm i`?

Reply:
Maybe. You should run `npm ci` instead of `npm i`

Reply:
First time I've heard of `npm ci`. Cool, I'll use that from now on!

Reply:
if you could revert this change that would be nice